docs/desktop-wpf/migration/convert-project-from-net-framework.md

```diff
@@ -120,107 +127,107 @@ Once the NuGet packages are updated to recent versions, the `<PackageReference>`
 
-To create a new csproj file for the Bean Trader sample, run `dotnet new wpf` in a temporary directory and move the generated csproj file into the BeanTraderClient folder and rename it BeanTraderClient.Core.csproj.
+To create a new project file for the Bean Trader sample, run `dotnet new wpf` in a temporary directory and move the generated *.csproj* file into the BeanTraderClient folder and rename it BeanTraderClient.Core.csproj.
 
 Because the new project file format automatically includes C# files, *resx* files, and XAML files that it finds in or under its directory, the project file is already almost complete! To finish the migration, open the old and new project files side-by-side and look through the old one to see if any information it contains needs to be migrated. In the Bean Trader sample case, the following items should be copied to the new project:
 
 * The `<RootNamespace>`, `<AssemblyName>`, and `<ApplicationIcon>` properties should all be copied.
 
-* We also need to add a `<GenerateAssemblyInfo>false</GenerateAssemblyInfo>` property to the new project file since the Bean Trader sample includes assembly-level attributes (like `[AssemblyTitle]`) in an AssemblyInfo.cs file. By default, new SDK-style projects will autogenerate these attributes based on properties in the csproj file. Because we don't want that to happen in this case (the autogenerated attributes would conflict with those from AssemblyInfo.cs), we disable the autogenerated attributes with `<GenerateAssemblyInfo>`.
+* You also need to add a `<GenerateAssemblyInfo>false</GenerateAssemblyInfo>` property to the new project file since the Bean Trader sample includes assembly-level attributes (like `[AssemblyTitle]`) in an AssemblyInfo.cs file. By default, new SDK-style projects will autogenerate these attributes based on properties in the csproj file. Because you don't want that to happen in this case (the autogenerated attributes would conflict with those from AssemblyInfo.cs), you disable the autogenerated attributes with `<GenerateAssemblyInfo>`.
```

Comment:
```suggestion
- You also need to add a `<GenerateAssemblyInfo>false</GenerateAssemblyInfo>` property to the new project file since the Bean Trader sample includes assembly-level attributes (like `[AssemblyTitle]`) in an AssemblyInfo.cs file. By default, new SDK-style projects will autogenerate these attributes based on properties in the csproj file. Because you don't want that to happen in this case (the autogenerated attributes would conflict with those from AssemblyInfo.cs), you disable the autogenerated attributes with `<GenerateAssemblyInfo>`.
```